### PR TITLE
Add recipe for qnmfits

### PR DESCRIPTION
**Pull request by eliotfinch**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by eliotfinch** [comment]
@conda-forge/help-python

I have a dependency (`scri`) which fails on Windows. If I want to use `noarch: python`, what is the best way to proceed?

**Comment by jaimergp** [comment]
I think that [`scri` can be `noarch`](https://github.com/moble/scri/blob/7d3bf3a3b6c90f427b6ce4c30374e58aec9938d0/pyproject.toml#L15). You can [open a PR in their feedstock](https://github.com/conda-forge/scri-feedstock/blob/main/recipe/meta.yaml). That said, that failure is not a blocker for your merge here. Once `scri` is available on Windows it'll just work for end users.

Let me review.

File: recipes/qnmfits/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - qnm >=0.4.3
+
+test:
+  imports:
+    - qnmfits
+  requires:
+    - python {{ python_min }}
+
+about:
+  home: https://github.com/sxs-collaboration/qnmfits
```
Comment (jaimergp) [inline recipes/qnmfits/meta.yaml:34]: This URL is giving 404 errors. Is it public?

Comment (eliotfinch) [reply recipes/qnmfits/meta.yaml:34]: The repository is now public and the link should work.

**Review by jaimergp — APPROVED** [review]

File: recipes/qnmfits/meta.yaml (hunk 1 above)
Comment (jaimergp) [reply recipes/qnmfits/meta.yaml:34]: Thanks!

**Pull request merged**